packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -632,16 +633,15 @@ async function waitUntilMessage(
 
     console.log("\n");
     console.log(
-      `${newRecordingIds.length} new recordings have been saved. Open each recording to ensure it has been pre-processed:`
-    );
-    console.log(
-      newRecordingIds
-        .map(recordingId => ` • ${blue(underline(`https://go/r/${recordingId}`))}`)
-        .join("\n")
+      `${newRecordingIds.length} new recordings have been saved. Loading each recording to ensure it has been pre-processed.`
     );
 
     const { exampleToTestMap } = getStats();
 
+    for (const recordingId of newRecordingIds) {
+      await loadRecording(recordingId);
+    }
```

Comment:
We could parallelize this:
```js
await Promise.all(
  newRecordingIds.map(loadRecording);
)
```

If it didn't put too much load on the backend. @cweld510

Reply:
Yeah, I didn't do that intentionally. Maybe we can do in batches instead. 🤔

Reply:
I'll defer.

Reply:
We use a simple queue in the test inbox server which you could re-use to control the parallelism. It's probably safe to start at 5 since these are such small replays.

https://github.com/replayio/test-inbox/blob/445925057671f00df20c087c490f9143566a7b2a/src/testInboxServer.ts#L19C7-L19C12